src/native_module/core.cljs

```diff
@@ -522,23 +547,37 @@
 
 (defn backup-disabled-data-dir
   []
-  (.backupDisabledDataDir ^js (utils)))
+  (if false ;; Check for not use of status-backend
+    (.backupDisabledDataDir ^js (utils))
+    status-backend.config/data-dir-path))
 
 (defn fleets
   []
   (.fleets ^js (status)))
 
 (defn keystore-dir
   []
-  (.keystoreDir ^js (utils)))
+  (if false
+    (.keystoreDir ^js (utils))
+    status-backend.config/keystore-dir-path))
 
 (defn log-file-directory
   []
-  (.logFileDirectory ^js (log-manager)))
+  (if false
+    (.logFileDirectory ^js (log-manager))
+    status-backend.config/log-dir-path))
 
 (defn init-status-go-logging
   [{:keys [enable? mobile-system? log-level log-request-go? callback]}]
-  (.initLogging ^js (log-manager) enable? mobile-system? log-level log-request-go? callback))
+  (if false
+    (.initLogging ^js (log-manager) enable? mobile-system? log-level log-request-go? callback)
+    (.initLogging ^js (log-manager)
+                  {:Enabled        enable?
+                   :MobileSystem   mobile-system?
+                   :Level          log-level
+                   :LogRequestGo   log-request-go?
+                   :LogRequestFile status-backend.config/log-request-file-path}
+                  callback)))
```

Comment:
Why do we call `.initLogging` on `(log-manager)` for both cases here?

Reply:
the new one calls status-backend in a compatible way

Reply:
oh does it change the destination of the logs to point to the status backend?

Reply:
Yes.